library/sonarqube

```diff
@@ -4,16 +4,16 @@ Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
              Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
 GitRepo: https://github.com/SonarSource/docker-sonarqube.git
-GitCommit: 2908701e165041cd2ffc9b618effb3345d4c60e9
+GitCommit: f5cdd43e0d6d931ca12becbac9a495536a3869dd
 
 Tags: 7.9.4-community, 7.9-community, lts
 Directory: 7/community
 
-Tags: 8.4.2-community, 8.4-community, 8-community, community, latest
+Tags: 8.5-community, 8-community, community, latest
```

Comment:
Did you still want to have the more specific tag for `8.5.0`?

Reply:
Yes we do. Thank you.

Reply:
Indeed. I've just pushed the suggested changes. Thanks